scripts/base.py

```diff
@@ -0,0 +1,80 @@
+import os
+import subprocess
+import sys
+import re
+
+ROOT = (
+    subprocess.check_output(["git", "rev-parse", "--show-toplevel"])
+    .decode("ascii")
+    .strip()
+)
+PROJECT_NAME = "Strawberry"
+
+# Files configuration
+
+RELEASE_FILE_NAME = "RELEASE.md"
+RELEASE_FILE = os.path.join(ROOT, RELEASE_FILE_NAME)
```

Comment:
nitpick, maybe it would be nicer to use [Pathlib](https://docs.python.org/3/library/pathlib.html) since we are on python 3 :)

Reply:
sure, will do it soon

Reply:
done!